ipwb/replay.py

```diff
@@ -975,6 +972,11 @@ def getCDXJLine_binarySearch(
         return lineFound
 
 
+def cdxjExists(pathOrHash):
```

Comment:
This is currently a dead code. Also, if a local index file is supplied and does not exist, I think the process will not be killed with the current work flow of code.

Reply:
Oops, did not mean to leave that in there. It has been removed in 3806ea3.

Reply:
...I just routed `start()` into existing logic and prevented an empty string from being returned in a scenario where we want to exit and inform the person running the script.

Reply:
Perhaps we can create a separate ticket for the IPFS scheme and merge this one for now.